resources/trusted_certificate.rb

```diff
@@ -79,16 +93,4 @@
   def update_cert_command
     platform_family?('debian', 'suse') ? 'update-ca-certificates' : 'update-ca-trust extract'
   end
-
-  # @return [String] the platform specific path to certs
-  def certificate_path
-    case node['platform_family']
-    when 'debian'
-      '/usr/local/share/ca-certificates'
-    when 'suse'
-      '/etc/pki/trust/anchors/'
-    else # probably RHEL
-      '/etc/pki/ca-trust/source/anchors'
-    end
-  end
```

Comment:
This should stay where this is but maybe be renamed as `certificate_default_dir`

Reply:
Done